src/app/auth/login/page.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import TextInput from '@/components/TextInput/index';
+import { H1, H4Centered } from '@/styles/text';
+import { ForgotPassword, FormDiv, QuestionsDiv, SixteenDiv } from '../styles';
+import supabase from '../../../api/supabase/createClient';
+
+export default function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  // const [errorMessage, setErrorMessage] = useState('');
+  const { push } = useRouter();
+  // commenting out Sign Up to use in the next PR -> create a separate sign in page
+  /*
+  const handleSignUp = async () => {
+    const { error } = await supabase.auth.signUp({
+      email,
+      password,
+    });
+
+    if (error) {
+      throw new Error(`An error occurred trying to sign up: ${error.message}`);
+    }
+    push('/');
+  };
+  */
+
+  const handleSignIn = async () => {
+    const { error } = await supabase.auth.signInWithPassword({
+      email,
+      password,
+    });
+
+    if (error) {
+      throw new Error(`An error occurred trying to sign in: ${error.message}`);
+    }
+    push('/');
+  };
+
+  return (
+    <FormDiv>
+      <H1>Log In</H1>
+      <SixteenDiv>
```

Comment:
mmmm, the problem is that the gap in the main div is 40 px, so I would still need to make a div for the 2 questions + the Forgot Password. I think it's best to keep the div?

Reply:
ahh I see, perhaps instead of using gap in the big div, we add vertical margin of 40px to the `QuestionDiv`? This is really minor and only saves us one `div` nesting, so feel free to decide whether or not to implement that!

Reply:
yea i agree, might be a lil cleaner without the `SixteenDiv` but i kinda like the use of gap over margin here so up to you! as jinkang said, it's pretty minor